src/utils/components/NetworkInterfaceModal/utils/helpers.ts

```diff
@@ -128,11 +130,15 @@ export const createInterface = (
   interfaceMACAddress: string,
   interfaceType = interfacesTypes.bridge,
 ): V1Interface => {
+  const resolvedInterfaceProp = interfaceLabels[interfaceType];
+  const validInterfaceProp: keyof V1Interface =
+    resolvedInterfaceProp === UDN_BINDING_NAME ? 'bridge' : resolvedInterfaceProp;
```

Comment:
I'm pretty sure there's a const or an enum for 'bridge', can you please check that? if missing add a const for it please

Reply:
No existing constant/enum found. I've found however a duplicated interface mapping with similar scope - https://github.com/kubevirt-ui/kubevirt-plugin/blob/5aab5d3fd6ad08ffa0a0ab637f99be12f18a648c/src/views/catalog/wizard/tabs/overview/components/WizardOverviewNetworksTable/WizardOverviewNetworksTable.tsx#L10